code-challenges.js

```diff
@@ -10,45 +10,154 @@
+// This is another question where I knew the answer and then I started typing and getting ahead of myself and not really visualizing how the code was going to look and in what order.
+// So they were a few stumbling blocks that I had to slow down and really think through it.
+// Then I saw that Sara One of the instructors were on and asked for help.
+// The only hint she gave me was to check my parentheses.
+// As I said before I was just in too big of a hurry and totally missed a set of parentheses in my consul log. 
+// Once I had fix that problem the code ran perfectly. 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to also accept non-number edge cases. Inform your user if the variable passed into the subtraction function is not a number.
 // Use the two sets of test variables provided below. Expected output: "Your input is not a number", 3
 // Hint: There is an operator in JavaScript called typeof :)
 
-var numberExampleRefactor1 = 42
-var numberExampleRefactor2 = "hello"
-
-var numberExampleRefactor3 = 27
-var numberExampleRefactor4 = 24
+// var numberExampleRefactor1 = 42
+// var numberExampleRefactor2 = "hello"
+
+// var numberExampleRefactor3 = 27
+// var numberExampleRefactor4 = 24
+
+// const equation1 = (answer) => {
+//     if(numberExampleRefactor1 > numberExampleRefactor2) {
+//         return numberExampleRefactor1 - numberExampleRefactor2
+//     } else if (numberExampleRefactor1 < numberExampleRefactor2) {
+//         return `${numberExampleRefactor1} is smaller then ${numberExampleRefactor2} so therefore I cannot compute`
+//     } else {
+//         return `Your input is not a ${typeof numberExampleRefactor1}`
+//     }
+// }
+// console.log(equation1())
+
+// const equation2 = (answer) => {
+//     if(numberExampleRefactor3 > numberExampleRefactor4) {
+//         return numberExampleRefactor3 - numberExampleRefactor4
+//     } else if (numberExampleRefactor3 < numberExampleRefactor4) {
+//         return `${numberExampleRefactor3} is smaller then ${numberExampleRefactor4} so therefore I cannot compute`
+//     }
+// }
+// console.log(equation2())
+/////////////////////////////////////////////////////////////////////// Todd's Pseudocode //////////////////////////////////////////////////////////////////
+// When I read through this the first time I was lost for about 20 minutes until I read up on typeof.
+// And because it took me that time I was able to slow down visualize how the code was going to look and what areas I needed to fix in order for it to display the right output.
+// Finding out that the command typeof basically gives you the data type of the input that you have put in.
+// If you put in a string, boolean, object or even a funcation it will show you.
```

Comment:
Great work researching typeof-- next time I would like to see the implementation of what you researched so I can geek out with you on it.